.evergreen/config.yml

```
@@ -834,6 +834,8 @@ buildvariants:
   matrix_spec: { "php-edge-versions": "latest-stable", "driver-versions": "latest-stable" }
   display_name: "Atlas Data Lake"
   run_on: debian11
+  expansions:
+    VARIANT: debian11
```

```suggestion
     VARIANT: debian11 # Referenced by ADL build script for downloading MQLRun
```

As originally suggested in https://github.com/mongodb/mongo-php-library/pull/994#discussion_r998357960